### PR TITLE
Fixed missing host details for capacity hosts and also fixed env name

### PR DESCRIPTION
**Pull request by yujunglo**
@sbaogang @jinruh 
(1) Fixed missing host details for capacity hosts.
(2) Fixed wrong env name in hosts details.

### CODE REVIEW

**Comment by sbaogang** [comment]
r+

**Comment by jinruh** [comment]
It would be nice to put this filtering on the client side to make the API more general.

**Comment by yujunglo** [comment]
Updated w/ @jinruh comment.

File: deploy-service/common/src/main/java/com/pinterest/deployservice/db/DBHostDAOImpl.java [hunk 1]
```diff
@@ -209,6 +211,6 @@ public HostBean getByEnvIdAndHostId(String envId, String hostId) throws Exceptio
     @Override
     public HostBean getByEnvIdAndHostName(String envId, String hostName) throws Exception {
```
Comment (jinruh) [inline deploy-service/common/src/main/java/com/pinterest/deployservice/db/DBHostDAOImpl.java]: Do we still need this function call?

Comment (yujunglo) [inline deploy-service/common/src/main/java/com/pinterest/deployservice/db/DBHostDAOImpl.java]: Yes, this is used in EnvironHandler.

**Pull request merged**